lively.merger/merger.js

```diff
@@ -118,3 +118,30 @@ export async function mergeMorphsWithIdsIntoA (morphAid, morphBid) {
 export async function mergeMorphsWithIdsIntoB (morphAid, morphBid) {
   return mergeMorphsWithIdsIntoA(morphBid, morphAid);
 }
+
+export async function mergeWorlds (expectedVersion, actualVersion, strategy) {
```

Comment:
are expected and actual really good names for this? Maybe something like "storedVersion" and "currentlyLoadedVersion" would fit better